src/libraries/System.Memory/ref/System.Memory.Forwards.cs

```diff
@@ -11,3 +11,4 @@
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Buffers.MemoryManager<>))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Buffers.OperationStatus))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Buffers.Text.Base64))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Buffers.Text.Base64Url))]
```

Comment:
Why is this needed?

Reply:
Ah, looks like it's not needed, added it because `System.Buffers.Text.Base64` above and thought its needed because System.Memory ref doesn't have it